babelize/cfg.py

```diff
@@ -16,12 +16,23 @@ def __init__(self, cfg):
     def root_dir(self):
         return Path(self.cfg['root_dir']).resolve()
 
+    @property
+    def translations(self):
+        return self.cfg['translations']
```

Comment:
Are these the expected translations? All ISO 3166-1 alpha-2 codes, I assume?

Reply:
Right. These are the ones supported by Hugo. For now I have assumed they will be provided in the config. In the future, it could optional as we could read Hugo's config file and get the current ones from there. However, for the time being I would like to separate what you have in Hugo and what you manage with babelize (so you could have 1 translation being managed by babelize but multiple ones in your Hugo site).

Reply:
Yeah agreed, we shouldn't be too tied to the Hugo implementation